### ISSUE TITLE
edit-bank doesn't change parent

### ISSUE THREAD

**Issue by ryanday36** [flux-framework/flux-accounting#298]
I was just asked to move an existing bank on corona under a new parent bank. It doesn't look like the `edit-bank` command can change anything other than shares, so I deleted the bank and re-created it under the new parent. I also had to delete and re-create all of the users of the bank. Can the `edit-bank` command be extended to moving banks to a new parent?

**Comment by cmoussa1** [comment]
Yeah, I think that definitely should be possible. I'll try to work on that to get that in before our next release. Sorry that didn't exist before, I didn't think of it!
